plugins/modules/ipaconfig.py

```diff
@@ -415,38 +414,33 @@ def main():
                     except ipalib_errors.EmptyModlist:
                         changed = False
         else:
-            rawresult = ansible_module.ipa_command_no_name("config_show", {})
-            result = rawresult['result']
             del result['dn']
+            type_map = {"str": str, "int": int, "list": list, "bool": bool}
```

Comment:
"bool" in type_map is not used.

Reply:
It might be used in the last line of the data adaption, to cast the type of `value` (`exit_args[k] = type_map[arg_type](value)`).